Add changelog.md in main and develop branch (#8)
XgridInc/xc3: CHANGELOG.md

# Changelog

All notable changes to the `XgridInc/xc3` repository will be documented in this file.

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [1.0.0] - 2023-05-11

### Added
- Initial release of the `XgridInc/xc3` repository.

> - `cloud_custodian_policies` folder which has the yaml for all the cloud custodian policies

Reviewer: - `cloud_custodian_policies` folder containing the YAML file for all the cloud custodian policies

Author: Resolved in [bea9bfe](https://github.com/XgridInc/xc3/pull/8/commits/bea9bfefc5c3042b6a9437670d0ac2db8d12b89d)

> - `cloud_custodian_policies` folder which has the yaml for all the cloud custodian policies

Reviewer 2: ```suggestion
- `cloud_custodian_policies` folder having cloud custodian policies for tagging compliance, monitoring and alerting
```

- `Pre-requirements` folder to refer the IAM Permission Set required to setup XC3

> - `Infrastructure` folder with the IaC necessary to provision XC3

Reviewer 2: ```suggestion
- `Infrastructure` folder with the necessary IaC to provision XC3 infrastack
```

> - `src` folder which has the code for all lambda functions

Reviewer 2: ```suggestion
- `src` folder having the XC3 backend
```

> - `tests` folder to test your IaC implementation and contribution

Reviewer 2: make this testing version to v1.0.0

Author: Resolved in [bea9bfe](https://github.com/XgridInc/xc3/pull/8/commits/bea9bfefc5c3042b6a9437670d0ac2db8d12b89d)

> - `tests` folder to test your IaC implementation and contribution

Reviewer 2: ```suggestion
- `tests` folder with unit testing implementation v1.0.0 for terraform infrastructure.
```

> - `workflows` folder which contains the diagram of all the workflows implemented

Reviewer 2: ```suggestion
- `workflows` folder which contains the diagram of all the XC3 workflows implemented
```

- `CODE_OF_CONDUCT.md` file for the Code of Conduct for the community

> - `CONTRIBUTING.md` file which contains the detail on how to contribute to the Project

Reviewer 2: ```suggestion
- `CONTRIBUTING.md` file which contains the detail of how to contribute to the Project
```

> - `ROADMAP.md` file which has the information what is currently implemented and what are the future plans

Reviewer: - `ROADMAP.md` file which has the information around what is currently implemented and what are the future plans

Author: Resolved in [bea9bfe](https://github.com/XgridInc/xc3/pull/8/commits/bea9bfefc5c3042b6a9437670d0ac2db8d12b89d)

> - `ROADMAP.md` file which has the information what is currently implemented and what are the future plans

Reviewer 2: ```suggestion
- `ROADMAP.md` file which has information about XC3's existing features and its future plans
```

- `UserGuide.md` file provides the guide on how to use XC3 Grafana Dashboards effectively

### Changed

> - Updated the startup script

Reviewer 2: ```suggestion
- Updated the startup script of XC3's EC2 instance 
```
